### PR TITLE
legacy: BibEditMulti removal

### PR DESCRIPTION
**Pull request by hachreak**
* INCOMPATIBLE Removes legacy BibEditMulti module. (addresses #3233)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio/modules/access/local_config.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
```
Comment (jirikuncar) [inline invenio/modules/access/local_config.py]: @hachreak please do only minimal changes here. Thanks

Comment (hachreak) [inline invenio/modules/access/local_config.py]: it's only for pep8/257

**Comment by jirikuncar** [comment]
:shipit:

**Comment by hachreak** [comment]
ping @jirikuncar

**Pull request merged**